macros/src/utils.rs

```diff
@@ -97,7 +97,7 @@ pub fn raw_name_to_ts_field(value: String) -> String {
 }
 
 /// Parse all `#[ts(..)]` attributes from the given slice.
-pub fn parse_attrs<'a, A>(attrs: &'a [Attribute]) -> Result<A>
+pub(crate) fn parse_attrs<'a, A>(attrs: &'a [Attribute]) -> Result<A>
```

Comment:
In lower versions of Rust, this would complain about leaking a crate-private trait